content/kubermatic/main/references/reference_setup/_index.en.md

```diff
@@ -0,0 +1,478 @@
+
+  - VNet with attached Subnet(s) with a big enough IP range for all planned VMs (management or/and user cluster machines):
+
+    - for a small setup,/24 should be enough
+
+    - subnet with expected routing needs to be set as well
+
+- Azure Cloud Resources managed by [Kubernetes Azure Cloud Provider](https://github.com/kubernetes-sigs/cloud-provider-azure)
+
+  - Kubermatic will create on-demand resources like volumes and LoadBalancer, and ensure the permissions and infrastructure setup are enabled for it, see [SIG - Cloud Provider Azure - Permissions](https://kubernetes-sigs.github.io/cloud-provider-azure/topics/azure-permissions/)
+
+  - Also, ensure that API is reachable from the seed and worker node network, see [Azure API Docs](https://docs.microsoft.com/en-us/rest/api/azure/)
+
+- Azure Machine Deployment information:
+
+  - If additional Seed at Azure is planned
+
+    - Ensure for the KubeOne seed the initial terraform module can be applied [Azure Quickstart Terraform configs](https://github.com/kubermatic/kubeone/tree/master/examples/terraform/azure) by using the [Terraform Azure Provider](https://www.terraform.io/docs/providers/azurerm/index.html)
+
+  - Ensure to share the needed parameter of [Azure - machine.spec.providerConfig.cloudProviderSpec](https://github.com/kubermatic/machine-controller/blob/main/docs/cloud-provider.md#azure)
+
+
+### **Integration Option to existing KKP**
+
+#### Option I - Workers only in Azure
+
+![one seed](integration-1.png)
+
+- Worker Node IP Range needs to reach seed cluster user control plan endpoints, see [Expose Strategy](https://docs.kubermatic.com/kubermatic/main/tutorials-howtos/networking/expose-strategies/)
+
+- KKP will create then a secure GRPC tunnel via [Kubernetes Konnectivity proxy](https://docs.kubermatic.com/kubermatic/main/architecture/concept/kkp-concepts/networking/#kubernetes-konnectivity-proxy) between worker and control plan, so no way back connection needs to be opened
+
+- Existing seed cluster(s) need to reach [Azure API endpoints](https://docs.microsoft.com/en-us/rest/api/azure/) for the machine provisioning
+
+- Application traffic gets exposed at Azure workers (Cloud LBs)
+
+
+#### Option II - Additional Seed at Azure + Worker in Azure
+
+![multi seed](integration-2.png)
+
+- The Seed Cluster Kubernetes API endpoint at Azure needs to be reachable
+
+  - Azure Load Balancer with https endpoint, managed Azure CCM
+
+- Worker Node IP Range needs to talk to an additional node-port load balancer at the Azure seed cluster
+
+  - Additional DNS entry for Azure seed is needed e.g. \*.azure-seed.kubermatic.example.com
+
+  - Workers <-> Seed communication is in the Azure Net only
+
+- User Cluster users need to reach the Azure seed cluster load balancer node-port
+
+  - IP/DNS of Azure LoadBalancer
+
+- Application traffic gets exposed to Azure workers (Cloud LBs managed by Azure CCM)
+
+- Host for Seed provisioning (KubeOne setup) needs to reach the Azure network VMs by SSH
```

Comment:
we should split this in subpages or Tabs

Reply:
@toschneck Later can be an improvement, but let's not make this a blocker for merging this PR to the docs

Reply:
this never get done then